src/components/ContactBanner.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [Optional] Kindly don' disable eslint configuration, it will lead you to have clear and professional code 👍